packages/story-editor/src/components/canvas/framesLayer.js

```diff
@@ -163,9 +186,36 @@ function FrameElements() {
 }
 
 function FramesLayer() {
+  const canvasId = useState(uuidv4)[0];
+  const canvasRef = useRef();
+  const enterFocusGroup = useEditLayerFocusManager(
+    ({ enterFocusGroup }) => enterFocusGroup
+  );
+
+  useKeyDownEffect(
+    canvasRef,
+    { key: ['enter'] },
+    () => {
+      enterFocusGroup({
+        groupId: FOCUS_GROUPS.EDIT_CANVAS,
+        cleanup: () => canvasRef.current?.focus(),
+      });
+    },
+    [enterFocusGroup]
+  );
+
   return (
     <FramesNavAndSelection>
-      <FrameElements />
+      <FocusContainer
+        id={canvasId}
+        data-testid="canvas-focus-container"
+        ref={canvasRef}
+        aria-label={FOCUS_CONTAINER_MESSAGE}
+        // eslint-disable-next-line styled-components-a11y/no-noninteractive-tabindex -- Container used to separate elements from normal tab order.
+        tabIndex={0}
+      />
+      {/* Hide canvas from normal tab flow. Use focus group to focus canvas */}
+      <FrameElements role="main" aria-labelledby={canvasId} />
```

Comment:
🤔  Seems like the main content area of the editor is the canvas - so the [`main` role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/main_role) felt like the correct role to put here.

Thoughts? [`region` seemed to vague](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/region_role), and [`application` seemed too complex](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/application_role) since it tells screen readers that "no traditional HTML interpretation techniques should be used"

Reply:
hmmm. #wp-body has the main role, so this would nest a main role in a main role. the display and frames layers are regions - those both live inside the canvas. 

I vote this is a region. 

Also, having this be a `main` causes aXe to get angry about multiples: https://github.com/GoogleForCreators/web-stories-wp/runs/5858290694?check_suite_focus=true#step:9:575